src/main/scala/firrtl/stage/phases/AddCircuit.scala

```diff
@@ -0,0 +1,55 @@
+package firrtl.stage.phases
+
+import firrtl.stage._
+
+import firrtl.{AnnotationSeq, Parser, proto}
+import firrtl.annotations.DeletedAnnotation
+import firrtl.options.{OptionsException, Phase}
+
+import java.io.FileNotFoundException
+
+object AddCircuit extends Phase {
+
+  /** Extract the info mode from an [[AnnotationSeq]]
+    * @param annos some annotations
+    * @return the info mode
+    * @note This will only work properly for an [[AnnotationSeq]] that passes [[checkAnnotations]]
```

Comment:
checkAnnotations here does not seem to be imported, for scaladoc link to work

Reply:
Should be `Checks`. Fixed.